src/BlazorQuery.TestApp.CommonLib/BlazorQuery.TestApp.CommonLib.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk.Razor">
```

Comment:
What is the idea of renaming the Project to CommonLib?

Reply:
I have split the TestApp projet in 3 projects. I have follow the principles of the course https://www.pluralsight.com/courses/getting-started-blazor : 1 project for blazor webassembly, 1 project for blazor server, and 1 project that contains all the common code. So CommonLib is the code shared by the 2 projects...